src/snowflake/snowpark/_internal/analyzer/select_statement.py

```diff
@@ -474,7 +482,11 @@ def __init__(
             self._schema_query = sql
             self._query_param = params
 
-    def __deepcopy__(self, memodict={}) -> "SelectSQL":  # noqa: B006
+    def __deepcopy__(self, memodict=None) -> "SelectSQL":  # noqa: B006
+        if memodict is None:
+            memodict = {}
```

Comment:
what is the benefit to have None here? does it do the same thing as passing memodict={}?

Reply:
I guess you are right. they are the same